src/popup/SLStorage.js

```diff
@@ -48,7 +50,7 @@ class SLStorage {
   }
 
   static setTemporary(key, value) {
-    TEMP[key] = value;
+    TEMP[key] = Utils.cloneObject(value);
```

Comment:
ah why do we need to do a deep clone here?

Reply:
That's because in `Main.vue`, we have `SLStorage.setTemporary("userMailboxes", this.mailboxes);`

Here, `this.mailboxes` is an object managed by vue. Vue actually injected its getter & setter into the object, so to use it elsewhere, we need to make a clone without these setter/getter.

Reply:
I see, that makes sense indeed!